frontend/src/Editor/Components/Table/Table.jsx

```diff
@@ -155,7 +155,7 @@ export function Table({
 
   const defaultColumn = React.useMemo(
     () => ({
-      minWidth: 60,
+      minWidth: 10,
```

Comment:
This looks a bit aggressive for the width of other column types. Can we figure out a better way to reduce the image size instead of changing the default column width, which may affect other column types?